### DISCUSSION TITLE
How soon will Xcode 13.3 be available?

### DISCUSSION THREAD

**Discussion by rhondadevore** [actions/virtual-environments#5370] [Installed software]
As you know, anyone who uses swift packages is forced to use a workaround? I realize you need to update to macOS 12 in order to install the newest Xcode, but was wondering if there is an eta on having this available?

**Comment by miketimofeev** [comment]
Hi @rhondadevore! Thank you for reaching us. macOS-12 is already available as a private preview. Please feel one of these form depends on which platform you use:
- Azure DevOps https://forms.gle/3DzreCjz4V2DR8aB7
- GitHub actions https://forms.gle/D1ihWjw3xDfjYbu67

**Comment by mqln** [comment]
@miketimofeev Hiya, is there an eta for when macOS-12 will be moved out of private?

**Reply by miketimofeev** [reply]
@mqln it is already for GitHub, for AzDO most likely in late May or June
